content/blog/2019/06/gsoc_yufei.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "GSoC Project Progress : Cloud features for external workspace manager"
+tags:
+- jenkins
+- gsoc
+author: imaffe
+description: "Short summary of coding period 1" # optional
+opengraph:
+  image: /images/folder/icon.png # optional
```

Comment:
This should either point to a valid image or the whole `opengraph` setting can be removed.